integration/spark/app/src/main/java/io/openlineage/spark/agent/OpenLineageSparkListener.java

```diff
@@ -312,16 +313,18 @@ private ArgumentParser parseConf(SparkConf conf) {
       Optional<Double> timeout = findSparkConfigKeyDouble(conf, SPARK_CONF_TIMEOUT);
       Optional<String> apiKey =
           findSparkConfigKey(conf, SPARK_CONF_API_KEY).filter(str -> !str.isEmpty());
+      Optional<String> overwriteName = 
+              findSparkConfigKey(conf, SPARK_CONF_OVERWRITE_NAME).filter(str -> !str.isEmpty());
       Optional<Map<String, String>> urlParams =
           findSparkUrlParams(conf, SPARK_CONF_URL_PARAM_PREFIX);
 
       boolean consoleMode =
           findSparkConfigKey(conf, SPARK_CONF_CONSOLE_TRANSPORT)
-              .map(v -> Boolean.valueOf(v))
+              .map(Boolean::valueOf)
               .filter(v -> v)
               .orElse(false);
       return new ArgumentParser(
```

Comment:
Consider using builder for `ArgumenetParser`.

Reply:
done